From e5b2e15583c6320ac093d69a3986f6fd455accff Mon Sep 17 00:00:00 2001
From: Daniele Nicolodi <daniele@grinta.net>
Date: Wed, 19 Apr 2023 15:34:14 +0200
Subject: [PATCH] TST: fix test when not executed from git checkout

Fixes #397.
---
 tests/test_project.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/tests/test_project.py b/tests/test_project.py
index 049e2e9..f0c9ed7 100644
--- a/tests/test_project.py
+++ b/tests/test_project.py
@@ -19,7 +19,7 @@ import pytest
 
 import mesonpy
 
-from .conftest import chdir, package_dir
+from .conftest import chdir, in_git_repo_context, package_dir
 
 
 @pytest.mark.parametrize(
@@ -92,8 +92,9 @@ def test_user_args(package_user_args, tmp_path, monkeypatch):
         'install-args': ('cli-install',),
     }
 
-    mesonpy.build_sdist(tmp_path, config_settings)
-    mesonpy.build_wheel(tmp_path, config_settings)
+    with in_git_repo_context():
+        mesonpy.build_sdist(tmp_path, config_settings)
+        mesonpy.build_wheel(tmp_path, config_settings)
 
     # check that the right commands are executed, namely that 'meson
     # compile' is used on Windows rather than a 'ninja' direct
-- 
2.40.0

